Casks/c/chatgpt-atlas.rb

```diff
@@ -0,0 +1,32 @@
+cask "chatgpt-atlas" do
+  version "1.2025.288.12,20251021143446000"
+  sha256 "0e91940abcd4d24fa80b7c0cde70f19e58d378934e7105f4ae53e14b248828a8"
+
+  url "https://persistent.oaistatic.com/atlas/public/ChatGPT_Atlas_Desktop_public_#{version.csv.first}_#{version.csv.second}.dmg",
+      verified: "persistent.oaistatic.com/atlas/public/"
+  name "ChatGPT Atlas"
+  desc "OpenAI's official browser with ChatGPT built in"
+  homepage "https://chatgpt.com/atlas"
+
+  livecheck do
+    url "https://persistent.oaistatic.com/atlas/public/sparkle_public_appcast.xml"
+    strategy :sparkle
+  end
+
+  auto_updates
```

Comment:
I think you accidentally a word @p-linnane 

```suggestion
  auto_updates true
```